src/renderers/shaders/ShaderChunk/begin_vertex.glsl

```diff
@@ -1,2 +1,28 @@
 
+#ifndef INSTANCE_TRANSFORM
+
 vec3 transformed = vec3( position );
+
+#else
+
+#ifndef INSTANCE_TRANSFORM_DEFINED
```

Comment:
Ok, so it was because of this:

[lambert](https://github.com/mrdoob/three.js/blob/e7dc951e829bad80c244001e6c63023e58ad8260/src/renderers/shaders/ShaderLib/meshlambert_vert.glsl#L35) and [phong](https://github.com/mrdoob/three.js/blob/e7dc951e829bad80c244001e6c63023e58ad8260/src/renderers/shaders/ShaderLib/meshphong_vert.glsl#L34) have `begin_vertex` happening after `defaultnormal_vertex`. 

While [depth_vertex](https://github.com/mrdoob/three.js/blob/6c7f000734f8579da37fb39e5c2e9e5e2dfb14f8/src/renderers/shaders/ShaderLib/depth_vert.glsl) doesn't use the normal chunks at all. 

It kinda made sense to define this matrix in the chunk where it's used, but the most convenient place might be `uv_pars_vertex`. Common is included in both vert and frag.